File: web3rpc/rpc-specs/api/code-samples/python/admin/startWS.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

adminHost = "127.0.0.1"
port = 8552
cors = ""
apis = "klay"

sdk = OpenSDK(host)
admin_response = sdk.admin.start_ws(adminHost, port, cors, apis)

print(admin_response)

File: web3rpc/rpc-specs/code-samples/javascript/admin/startWS.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const host = "127.0.0.1";
  const port = 8552;
  const cors = "";
  const apis = "klay";

  provider.admin
    .startWS({ host, port, cors, apis }, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
